python/servo/testing_commands.py

```diff
@@ -254,8 +252,9 @@ def test_content(self):
         help="Check all files, and run the WPT lint in tidy, even if unchanged",
     )
     @CommandArgument("--no-progress", default=False, action="store_true", help="Don't show progress for tidy")
-    def test_tidy(self, all_files, no_progress):
-        tidy_failed = tidy.scan(not all_files, not no_progress)
+    @CommandArgument("--report-ci", default=False, action="store_true", help="Put the lint result on the file")
```

Comment:
The `help` message needs to be updated. See https://github.com/servo/servo/pull/37294#discussion_r2147905263